server/plugin/command.go

```diff
@@ -323,3 +314,28 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 	p.postCommandResponse(args, fmt.Sprintf("Unknown action %v", action))
 	return &model.CommandResponse{}, nil
 }
+
+// parseCommand parses the entire command input string and retrieves the command, action and parameters
+func parseCommand(input string) (command, action string, parameters []string) {
+	// do not split if the whitespace is in between 2 quotes, e.g. "Hello World"
+	foundQuote := false
```

Comment:
Nit pick: If `foundQuote` is only used inside the anonymous function for `FieldsFunc` it should be scoped to only that function.

Reply:
Taking a look at the code, it seems that foundQuote needs to be outside, since you want to keep the "state" between different calls to the check function.